sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -82,6 +84,7 @@ def _create_attribute(self, **kwargs):
         self._error_policy = self._amqp_transport.create_retry_policy(self._config)
         self._name = kwargs.get("client_identifier") or f"SBSender-{uuid.uuid4()}"
         self._max_message_size_on_link = 0
```

Comment:
why do we set these both twice?

Reply:
is `create_attribute` called elsewhere ? otw yeah might be a mistake